### ISSUE TITLE
hints.char with space char

### ISSUE THREAD

**Issue by f380cedric** [qutebrowser/qutebrowser#5084]
**Version info (see `:version`)**:
https://paste.the-compiler.org/view/88874410
**Does the bug happen if you start with `--temp-basedir`?**:
Yes
**Description**
Would like to use space for hints. The key sequence is discarded when space is pressed.
**How to reproduce**

1. Add " " (space char) in hints.char (qute:settings).
2. press f or F on a webpage with link and try it.

**Comment by user202729** [comment]
Currently it isn't filtered correctly; but even if it's, it cannot be followed (because of https://github.com/qutebrowser/qutebrowser/issues/4368)

**Comment by f380cedric** [comment]
New version, new behavior
### version 
https://paste.the-compiler.org/view/b912fcbc

### behavior
The sequence works, even if there is space in it.
However, the hint box disappears after pressing space, so you have to remember it (not an issue if space is the last one ofc).

**Comment by The-Compiler** [comment]
Yep, that's due to b25c7d5b36a239f8520670390692c96a2812d300 fixing part of this issue, but it looks like more is needed to fully fix it.
